commands/export.go

```diff
@@ -0,0 +1,113 @@
+package commands
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/git-bug/git-bug/commands/execenv"
+	"github.com/git-bug/git-bug/entity"
+	"github.com/git-bug/git-bug/entity/dag"
+	"github.com/git-bug/git-bug/query"
+)
+
+type exportOptions struct {
+	actorQuery       []string
+	authorQuery      []string
+	labelQuery       []string
+	noQuery          []string
+	participantQuery []string
+	queryOptions     []string
+	statusQuery      []string
+	titleQuery       []string
+	sortBy           string
+	sortDirection    string
+}
+
+func newExportCommand(env *execenv.Env) *cobra.Command {
+	options := exportOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "export [QUERY]",
+		Short: "Export bugs as a series of operations.",
+		Long: `Export bugs as a series of operations.
+
+The output format is NDJSON (Newline delimited JSON).
+
+You can pass an additional query to filter and order the list. This query can be expressed either with a simple query language, flags, a natural language full text search, or a combination of the aforementioned.`,
+		Example: `See ls`,
```

Comment:
This should be an example of running the command, e.g.:

```suggestion
		Example: `git bug bug export`,
```